packages/server/src/plugin/landingPage/default/index.ts

```diff
@@ -133,7 +133,10 @@ id="embeddableExplorer"
 `;
 };
 
-const getEmbeddedSandboxHTML = (version: string, config: LandingPageConfig) => {
+export const getEmbeddedSandboxHTML = (
```

Comment:
In AS4, our imports are set up a bit differently.

Now, to import this plugin, you would:

`import ... from '@apollo/server/plugin/landingPage/default';`

This means that exports from this `index.ts` file that we're in are part of the public API (they can be imported by users now), and I'd prefer if we don't make that the case. Can we move these to an adjacent file(s)? i.e. `getEmbeddedSandboxHTML.ts` and `getEmbeddedSandboxHTML.ts` so they aren't exported from the `index.ts` _but_ are still accessible internally and for tests?